src/core/sys/posix/arpa/inet.d

```diff
@@ -199,23 +199,4 @@ NOTE: The following must must be defined in core.sys.posix.arpa.inet to break
 INET6_ADDRSTRLEN // from core.sys.posix.netinet.in_
 */
 
-version( linux )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
-else version( OSX )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
-else version( FreeBSD )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
-else version( Solaris )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
-else version( Android )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
+enum INET6_ADDRSTRLEN = 46;
```

Comment:
If you made a technical argument, that an IP6 address can't be longer than 46 characters or had shown the declaration in glibc (https://sourceware.org/git/?p=glibc.git;a=blob;f=inet/netinet/in.h;hb=98734cc50153c80047f4ed9c6772bc7e1e68c9f7#l233) which actually implements inet_ntop, then you could have convinced someone that the Linux declaration is wrong.